Sources/StreamChat/APIClient/Endpoints/Payloads/RawJSON.swift

```diff
@@ -4,10 +4,16 @@
 
 import Foundation
 
+public typealias CustomData = [String: RawJSON]
```

Comment:
`ExtraData`is not a good option, we used that before and it will make things super confusing if you upgrade (also it was not really an option during the refactoring).

I think it makes sense to keep the alias in place, makes it easier to change things in the future without breaking things

Reply:
if that's the case, we should rename all instances of `extraData` within SDK to `customData. `extraData: CustomData` seems wrong

Reply:
I rather rename `CustomData` into `Map`, we use `extraData` in Android